cypress/objects/TaskPage.js

```diff
@@ -1,3 +1,37 @@
-export const TaskPage = {
-    
-};
\ No newline at end of file
+class TaskPage {
+    visit() {
+        cy.visit("/tasks");
+    }
+    getCreateTaskButton() {
+        return cy.get('[type="button"]').contains('Create Task');
+    }
+    getTaskNameField() {
+        return cy.get('[id="taskDetailsForm.name"]');
+    }
+    getTaskDescriptionField() {
+        return cy.get('[id="taskDetailsForm.description"]');
+    }
+    getBillToClientDropdown() {
+        return cy.get('.Select-value');
+    }
+    getBillToClientOption() {
+        return cy.get('[aria-label="Yes"]');
+    }
+    getHourlyRateField() {
+        return cy.get('[id="taskDetailsForm.rate"]');
+    }
+    getSubmitButton() {
+        return cy.get('[type="submit"]');
+    }
+    assertUrl() {
```

Comment:
`assertUrl` doesn't say what kind of url it asserts. I suggest either passing what king of url you want assert, or renaming method to `assertNotCreateUrl` or something like that